dietpi/dietpi-software

```diff
@@ -4166,6 +4173,55 @@ WantedBy=multi-user.target
 _EOF_
 		fi
 
+		if To_Install 141 adsb-feeder # ADSB Feeder
+		then
+			# create the userdata directory and clone the adsb-feeder repo into /tmp
+			G_EXEC mkdir -p /mnt/dietpi_userdata/adsb-feeder
+			G_EXEC_OUTPUT=1 G_EXEC git clone -b dietpi 'https://github.com/dirkhh/adsb-feeder-image.git' /tmp/adsb-feeder
```

Comment:
As Git is only used on install here, can we just download the archive and skip Git as dependency?
```suggestion
			Download_Install 'https://github.com/dirkhh/adsb-feeder-image/archive/refs/heads/dietpi.tar.gz' /tmp
```
Should download and extract it to `/tmp/adsb-feeder-image-dietpi` (appending the branch) and removes the archive automatically.

Ah, now I see you derive the version string via Git. Okay, if this cannot be extracted easily elsewhere, not a big problem.

Reply:
yes - that's part of making it possible for people to ask questions and me to know which exact version they are using, so I'd really love to keep the git checkout.